src/main.rs

```diff
@@ -386,33 +516,42 @@ impl Node {
                             .map(|i| format!("- \\\"{i}\\\"\\l"))
                             .collect::<String>())
             },
-            swallows = if self.swallows.is_empty() {
+            swallows = if self.swallows.is_empty() || settings.no_swallows {
                 "No swallows"
```

Comment:
Perhaps the whole swallows window could be hidden if there are none/they are disabled?

Reply:
(this one's not necessary for merging, I'm just throwing the idea here)